lms/envs/common.py

```diff
@@ -2268,6 +2268,7 @@
 STATIC_TEMPLATE_VIEW_DEFAULT_FILE_EXTENSION = 'html'
 
 SUPPORT_SITE_LINK = ''
+ACTIVATION_EMAIL_SUPPORT_LINK = 'https://support.edx.org/hc/en-us/articles/227340127-Why-haven-t-I-received-my-activation-email-'
```

Comment:
Please don't hardcode edx specific links into the defaults we ship to openedx users.
People will end up contacting our support staff about other edX installs.

This is what the edx-internal repo is for (trivially changing an empty value or default value to the edX specific one).